components/datetime/examples/work_log.rs

```diff
@@ -6,6 +6,9 @@
 
 #![no_main] // https://github.com/unicode-org/icu4x/issues/395
 
+use icu_benchmark_macros::{main_setup, static_setup};
+static_setup!();
```

Comment:
Optional: here and elsewhere: if you only use the macro one time, I have a slight preference for

```
icu_benchmark_macros::static_setup!()
```

and down below

```
icu_benchmark_macros::main_setup!()
```

and then remove the `use` line.

Reply:
I agree, this is nicer.